spec/services/calculators/disqualification_calculator_spec.rb

```diff
@@ -15,67 +14,22 @@
   let(:conviction_length) { nil }
   let(:conviction_length_type) { nil }
 
-  describe Calculators::DisqualificationCalculator::Youths do
-    let(:under_age) { GenericYesNo::YES }
-
-    describe "#expiry_date" do
-      context "with a lengthless than or equal to 2.5 years" do
-        let(:conviction_length) { 2 }
-        let(:conviction_length_type) { "years" }
-
-        it { expect(calculator.expiry_date.to_s).to eq((known_date + 30.months).to_s) }
-      end
-
-      context "with a lengthgreater than 2.5 years" do
-        let(:conviction_length) { 3 }
-        let(:conviction_length_type) { "years" }
-
-        it { expect(calculator.expiry_date.to_s).to eq((known_date + 36.months).to_s) }
-      end
-
-      context "without a length" do
-        let(:conviction_length) { "no_length" }
-
-        it { expect(calculator.expiry_date.to_s).to eq("2021-04-30") }
-      end
-
-      context "with an indefinite length" do
-        let(:conviction_length_type) { "indefinite" }
-
-        it { expect(calculator.expiry_date).to eq(ResultsVariant::INDEFINITE) }
-      end
+  describe "#expiry_date" do
+    context "without a length" do
+      it { expect(calculator.expiry_date.to_s).to eq("2018-10-31") }
```

Comment:
If you look at the old tests they were set up so that the expectation is "known_data + some.duration". I think that is clearer to show what the expectation is rather than just a date with no context.